tests/utils/helper.ts

```diff
@@ -114,6 +114,35 @@ export async function setupInitialContractState(
   // set the owner to the first wallet
   state.owner = owner;
 
+  // add some reserved names
+  const currentDate = new Date(); // Get current date
+  const sixMonthsLater = new Date(); // Create a new date object
+  const sixMonthsPrevious = new Date(); // Create a new date object
+  sixMonthsLater.setMonth(currentDate.getMonth() + 6);
+  sixMonthsPrevious.setMonth(currentDate.getMonth() - 6);
+  state.reserved = {
+    ['1']: {}, // no owner, doesnt expire
+    ['2']: {}, // no owner, doesnt expire
+    ['3']: {}, // no owner, doesnt expire
+    ['4']: {}, // no owner, doesnt expire
```

Comment:
i'm thinking we put the the reserved name lengths in a constant E.G. `MIN_ALLOWED_NAME_LENGTH = 4` rather than storing these lengths in the reserved name. we can add shorter names (like `www` to this object, and when we remove the `MIN_ALLOWED_NAME_LENGTH` or evolve away from it, we don't have to do anything with the state of these reserved keys.